Dockerfile

```diff
@@ -72,15 +77,27 @@ RUN apt-get update -qq \
     texlive-science \
     fontconfig \
     redis-tools
+# ENV INSTALL_DEPS \
+#     git
```

Comment:
It remains because I am planning to sort out which dependencies are install deps and which are really needed, so the former can be removed after they are not needed.  This will reduce the size of the resulting Docker image.

Reply:
I'd like to suggest you do a multi-stage build if feasible at that point.